docs/UserGuide.md

```diff
@@ -113,7 +113,7 @@ You can use the [`ma`](#marking-attendance-ma) command to mark a student as pres
 
 **Tip:**
 Use negative marking to quickly take attendance for a class. Since all students are initially assumed to be present,
-you can quickly the attendace of an entire class by only marking the absentees as absent.
+you can quickly the attendance of an entire class by only marking the absentees as absent.
```

Comment:
"you can quickly TAKE/MARK the attendance"

Reply:
Fixed! Please review!

please? pretty please? pretty please with a cherry? pretty please with a cherry and cream? pretty please with a cherry, cream, and sprinkles? pretty please with a cherry, cream, sprinkles and chocolate sauce? pretty please with a cherry, cream, sprinkles, chocolate sauce and nuts? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts and oreos? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts, oreos and caramel? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts, oreos, caramel and gold flakes? pretty please with a cherry, cream. sprinkles, chocolate sauce, nuts, oreos, caramel, gold flakes and a 50 foot tall ice sculpture?